scripts/ci/wait_tcp.py

```diff
@@ -12,7 +12,7 @@ def tcp_ping(port, timeout):
     while time.time() - now < timeout:
         try:
             with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
-                sock.connect(('0.0.0.0', port))
+                sock.connect(('127.0.0.1', port))
```

Comment:
This is a client-side tcp for testing purpose only. Leaving it 0.0.0.0 may be better?